openedx/features/edly/cookies.py

```diff
@@ -78,6 +78,7 @@ def _get_edly_user_info_cookie_string(request):
                 request.user,
                 CourseCreatorRole()
             ) if getattr(request, 'user', None) else False,
+            'group_name': request.user.groups.first().name if getattr(request, 'user', None) else '',
```

Comment:
@rehan99000 The case of multiple permission groups will be missed.. you are only adding the first group name, while a user can belong to multiple permission groups. Kindly address that.

Reply:
added all groups